packages/gensx-cloud-mcp/package.json

```diff
@@ -6,6 +6,9 @@
   "main": "./dist/cjs/index.cjs",
   "module": "./dist/esm/index.js",
   "types": "./dist/cjs/index.d.ts",
+  "bin": {
+    "gensx-cloud-mcp": "./dist/esm/index.js"
```

Comment:
Since this is a CLI package, we don't need to support dual ESM/CJS builds. See https://github.com/gensx-inc/gensx/blob/main/packages/gensx/package.json for an example of a package like that.

Reply:
Ok, removed all of the rollup and cjs stuff to match the CLI build.